pom.xml

```diff
@@ -0,0 +1,107 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>uk.ac.ebi.eva</groupId>
+  <artifactId>eva-omicsdi-export</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <java.version>1.8</java.version>
+    <mockito.version>2.7.22</mockito.version>
+  </properties>
+
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>1.5.3.RELEASE</version>
+  </parent>
+
+  <dependencies>
+    <dependency>
+      <groupId>uk.ac.ebi.ddi</groupId>
+      <artifactId>ddi-validator</artifactId>
+      <version>1.1</version>
+    </dependency>
+    <dependency>
+      <groupId>uk.ac.ebi.eva</groupId>
+      <artifactId>opencga-storage-core</artifactId>
+      <version>0.5.4</version>
+    </dependency>
```

Comment:
Is this dependency necessary only for `QueryResponse` and `QueryResult`? Wouldn't it be enough to declare `datastore` directly for a more lightweight executable?